src/main/java/oakbot/listener/DadJokeListener.java

```diff
@@ -22,7 +22,7 @@
 	private final String botName;
 
 	private final Pattern regex = Pattern.compile( //@formatter:off
-		"(?i)" +
+		"(?i)(?:^|@[^ ]+|:\\d+|[.?!])\\s*" +
```

Comment:
Would you mind formatting this new addition so it's on its own line in the code? I think we're good after that.